### PR TITLE
Update links to point to new wiki

### PR DESCRIPTION
**Pull request by GeckoEidechse**
We moved from GitBook to MkDocs so the links to the wiki should updated accordingly.

### CODE REVIEW

File: Northstar.Client/mod/scripts/vscripts/ui/atlas_auth.nut [hunk 1]
```diff
@@ -37,7 +37,7 @@ void function AtlasAuthDialog_Threaded()
 	if ( res.errorCode != "" )
 		dialogData.message += format( "\n\n%s", Localize( "#AUTHENTICATION_FAILED_ERROR_CODE", res.errorCode ) )
 
-	string link = "https://r2northstar.gitbook.io/r2northstar-wiki/installing-northstar/troubleshooting"
+	string link = "https://docs.northstar.tf/Wiki/installing-northstar/troubleshooting/"
```
Comment (JMM889901) [inline Northstar.Client/mod/scripts/vscripts/ui/atlas_auth.nut:40]: Is there a reason to add trailing slashes? doesn't matter though

**Review by JMM889901 — APPROVED** [review]
It is strangely annoying to me that the link uses capitalised Wiki
anyway :shipit:

File: Northstar.Client/mod/scripts/vscripts/ui/atlas_auth.nut (hunk 1 above)
Comment (GeckoEidechse) [reply Northstar.Client/mod/scripts/vscripts/ui/atlas_auth.nut:40]: MkDocs redirects to the page with the trailing slash if the called URL doesn't have one ^^

Comment (JMM889901) [reply Northstar.Client/mod/scripts/vscripts/ui/atlas_auth.nut:40]: Ah so it just removes a needless redirect? Fair enough

**Comment by GeckoEidechse** [comment]
> It is strangely annoying to me that the link uses capitalised Wiki

That's cause it takes the folder name for the URL. We can switch to non-capitalised `W`. You can make a PR [to change the folder name in docs repo](https://github.com/R2Northstar/NorthstarDocs/tree/main/docs) if you wanna change it ^^

**Pull request merged**